php-mode.el

```diff
@@ -1033,10 +1043,30 @@ PHP heredoc."
   "PHP Mode face used to highlight function names."
   :group 'php-faces)
 
+(defface php-function-caller-face '((t (:inherit default)))
+  "PHP Mode face used to highlight calling function names."
+  :group 'php-faces)
```

Comment:
@zonuexe I think I don't understand what it does, so a better description probably is required indeed :P 

Could you post a code snippet to show us the element which would be highlighted with this face?